app/api/chemotion/attachment_api.rb

```diff
@@ -65,12 +65,39 @@ def validate_uuid_format(uuid)
         requires :cnt_only, type: Boolean, desc: 'return count number only'
       end
       get do
-        current_user.container = Container.create(name: 'inbox', container_type: 'root') unless current_user.container
+        if current_user
+          inbox_container = Container.where(name: 'inbox', container_type: 'root', containable_id: current_user.id).take
+          unless inbox_container
+            inbox_container = Container.create!(name: 'inbox', container_type: 'root', containable_id: current_user.id)
+          end
+
+          if params[:cnt_only]
```

Comment:
@jhmegorei The param was there for a long time and I don't want to change the parameter since it was out of my scope for the PR and may introduce a new issue.